examples/CloudFunction/README.md

```diff
@@ -0,0 +1,121 @@
+# Terraform Stackdriver Aggregated Export to PubSub
```

Comment:
Please rename the example folder structure to lower-case style and better names:
`examples/CloudFunction/README.md` --> `examples/cloud_function/README.md`
`examples/CloudFunction/gsuite-exporter-cloud-function` --> `examples/cloud_function/code`
...